Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/View.purs

```diff
@@ -3252,8 +3304,9 @@ editButtontView push state =
   , stroke $ "1," <> state.data.config.confirmPickUpLocationBorder
   , cornerRadius if (os == "IOS") then 15.0 else 20.0
   , padding (Padding 10 6 10 6)
-  , margin $ MarginLeft 10 
+  , margin $ MarginLeft 10
   , accessibilityHint "Edit Pickup Location : Button"
+  , visibility $ boolToVisibility $ state.props.currentStage /= EditPickUpLocation
```

Comment:
create a local variable for this.

Reply:
this condition is not redundant... used in two different functions